### PR TITLE
Add launch script for simulation manipulator

### PR DESCRIPTION
**Pull request by knord94**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,7 +23,13 @@ RUN mkdir -p catkin_ws/src
 
 WORKDIR /home/catkin_ws/src/
 
-RUN git clone -b noetic-devel https://github.com/ROBOTIS-GIT/turtlebot3_simulations.git
+RUN git clone -b noetic-devel https://github.com/ROBOTIS-GIT/turtlebot3_simulations.git &&\
+    git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation.git &&\
+    git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation_simulations.git &&\
+    git clone https://github.com/ROBOTIS-GIT/open_manipulator_dependencies.git &&\
+    apt install -y ros-noetic-ros-control* ros-noetic-control* ros-noetic-moveit*
```
Comment (oysand) [inline Dockerfile:30]: Shoulld these be installed using dependency in our ros package?

File: config/house.cfg [hunk 2]
```diff
@@ -3,3 +3,5 @@ Y_POS=1.0
 Z_POS=0.0
 INITIAL_POSE_X=-3.0
 INITIAL_POSE_Y=1.0
+OPEN_MANIPULATOR_GUI=true
+OPEN_MANIPULATOR_RVIZ=false
```
Comment (oysand) [inline config/house.cfg:7]: Add newline

File: Dockerfile (hunk 1 above)
Comment (knord94) [reply Dockerfile:30]: Did not manage to install as dependency, potentially due to *-asterix?

Comment (oysand) [reply Dockerfile:30]: It seems that there are multiple packages needed from each group. Could we identify which packages are needed?

Comment (knord94) [reply Dockerfile]: The three packages `ros_control` , `ros_controllers` and `moveit` seems to be sufficient. Tested with the simple gui and with rviz. Both seem to work now.

File: README.md [hunk 3]
```diff
@@ -146,6 +146,26 @@ Missions can be posted to the robot through [ISAR](https://github.com/equinor/is
...
+## Simulation with manipulator
+
+The turtlebot can be equipped with a [manipulator](https://emanual.robotis.com/docs/en/platform/turtlebot3/manipulation/#manipulation) which also can be included in the simulations. Additional software must be installed for the manipulator simulation.
+
+```bash
+$ cd ~/catkin_ws/src/
+$ git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation.git
+$ git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation_simulations.git
+$ git clone https://github.com/ROBOTIS-GIT/open_manipulator_dependencies.git
+$ sudo apt install ros-noetic-ros-control* && ros-noetic-control* && ros-noetic-moveit*
```
Comment (oysand) [inline README.md:158]: Can this line be removed now?

**Review by oysand — APPROVED** [review]
LGTM, rebase on main to avoid conflict before merging

**Pull request merged**